src/components/CheckboxFacet/CheckboxFacet.js

```diff
@@ -34,17 +34,30 @@ export default class CheckboxFacet extends React.Component {
   };
 
   onMoreClick = (totalOptions) => {
+    const {
+      onFetch,
```

Comment:
Please consider renaming onFetch to onLoad
Also please add it to propTypes as well as onSearch

Reply:
Thanks, fixed.